lib/Ohmbrewer_Runtime_Settings.h

```diff
@@ -69,13 +74,32 @@ namespace Ohmbrewer {
          */
         bool isWifiOff() const {return !getWifiStatus();};
 
+        /**
+         * The current unit. True => C, False => F
+         * @returns True => C, False => F
+         */
+        bool isTempUnitCelcius() const {return _celcius;};
+
+
+        /**
+         * Sets the Temp Unit and immediately saves to EEPROM. True => C, False => F
+         * @param status Whether the Celcius or not. True => C, False => F
+         */
+        const void setTempUnitAndSave(const bool celcius);
+
     protected:
 
         /**
          * Whether the WiFi should be on (true) or off (false)
          */
         bool _wifiStatus;
 
+        /**
+         * Whether the Temp Unit should be Celcius (true) or Fahrenheit (false)
+         */
+        bool _celcius;
```

Comment:
_celsius

Instead of commenting on all of these, from here on out let's just assume you'll find and replace throughout the project :wink: